src/index.ts

```diff
@@ -288,6 +289,7 @@ export type { CustomTypeModelNumberField } from "./types/model/number"
 export type { CustomTypeModelSelectField } from "./types/model/select"
 export type { CustomTypeModelTimestampField } from "./types/model/timestamp"
 export type { CustomTypeModelGeoPointField } from "./types/model/geoPoint"
+export type { CustomTypeModelTableField } from "./types/model/table"
 
```

Comment:
I would alphabetize `CustomTypeModelTableField` and `CustomTypeModelGeoPointField` here, but it's not a big deal.